index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <nav id="navbar">
+        <ul>
+            <li>
+                <a href="#welcome-section">Welcome</a>
+            </li>
+            <li>
+                <a href="#projects">Check my projects</a>
+            </li>
+            <li> <a href="#contact">Contact me</a>
+            </li>
+        </ul>
+    </nav>
+    <section id="welcome-section">Welcome!
+        <H1> Welcome to my personal Portfolio</H1>
+    </section>
+    <section id="projects">
+        <p><h2> Here are some of my projects:</h2></p>

```

Comment:
I recommend to do not putting h tags inside p tags because h tags themselves are blocked and no need to nest them inside p tags.

Reply:
Very good comment here. I like the fact that you included a reason behind your suggestion. 

- But your comment can be better if you specify clearly `h2` instead of just `h` and is void of grammatical errors. Please use `h2` in place of `h`.